### DISCUSSION TITLE
How To install on Mac OSX

### DISCUSSION THREAD

**Discussion by yeop-sang** [rafaelmardojai/blanket#262] [Q&A]
I'm trying to run blanket in my imac. 
Then, I have a trouble in running with `Unable to autolaunch a dbus-daemon without a $DISPLAY for X11` error. 
I think that problem which occurred because my desktop OS is OSX not linux using window x.

What can I do to build without window x?

**Comment by codybrom** [comment]
Hi @yeop-sang, I built an [open-source Blanket-inspired Mac app called Blankie](https://github.com/codybrom/blankie) you might be interested in!
